public/assets/js/.eslintrc

```diff
@@ -7,3 +7,4 @@ rules:
     no-var: off
     object-shorthand: off
     prefer-arrow-callback: off
+    prefer-template: off
```

Comment:
Is there a way to disable this inline instead of globally? I prefer the `+` concatenation employed below in instances like that, but in general, would like eslint to bark in favor of templates.

Reply:
Templates are used where we use es6, i.e. not in public/js. This file is specific for the files in this folder.